### DISCUSSION TITLE
Liquidity and Incentives on the CMDX/AKT pool on Cswap.

### DISCUSSION THREAD

**Discussion by SM12981** [akash-network/community#196] [Governance Proposals]
Hello Akash community!

We would like to propose a partnership between AKT and Comdex to provide liquidity and incentives on the CMDX/AKT pool on Cswap. We believe that this partnership would be beneficial for both communities and would help to increase the usage and adoption of both AKT and Comdex tokens.

The below proposal requires a total of $50k AKT from the community fund. The proposal is as follows:

We would like to seed the CMDX/AKT pool with a total of $50k liquidity, which will be split equally between CMDX and AKT tokens. To achieve this, we propose a token swap of $12.5k worth of CMDX for $12.5k worth of AKT. The AKT team will seed $25k ($12.5k AKT+$12.5k CMDX) in the CMDX/AKT pool, and the Comdex team will do the same
Furthermore, we propose to incentivize the CMDX/AKT pool for 90 days with $25k worth of AKT tokens from the AKT community fund. We will match this with $25k worth of CMDX from our side.

We believe that this partnership between AKT and Comdex will be mutually beneficial and will help to increase the adoption and usage of both tokens. We hope that the AKT community will consider our proposal and support this initiative.

**Comment by VilSa9** [comment]
I support the proposed partnership between AKT and Comdex to provide liquidity and incentives on the CMDX/AKT pool on Cswap. This collaboration can benefit both communities, promoting growth and adoption of AKT and Comdex tokens.

By seeding the CMDX/AKT pool with $50k liquidity and offering incentives for 90 days, this partnership can foster a strong foundation for both tokens. The equal distribution of liquidity through a token swap ensures fairness in the collaboration.

This partnership can increase usage, adoption, and visibility of both tokens, contributing to a robust ecosystem. I encourage community members to consider and support this initiative, as it can pave the way for the continued growth and success of AKT and Comdex tokens.

**Comment by hewi333** [comment]
Hi there and thank you for posting a proposed partnership here in discussions.

We have a smaller community pool at the moment and $50k at today’s prices would be around ~15% of it. Our Special Interest Groups (SIGs) and Working Groups (WGs) have been laser focused on progressing the development and use of decentralized cloud computing resources. I’m not sure adding liquidity to another exchange will necessarily add to those objectives at this moment in time.

While we appreciate Comdex’s willingness to match 1:1 to seed liquidity for a pool, given market conditions this may not be the right time, but we’ll be happy to revisit when conditions improve in the coming months.
